app/models/task.rb

```diff
@@ -6,13 +6,15 @@ class Task < ActiveRecord::Base
   attr_accessible :name,:position,:status
   after_create :initialise_sharetask
   has_one :check_list
+  scope :active, -> { where(:status => false) }
+  scope :inactive, -> { where(:status=> true) }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.